srcStatic/StringConversion.cpp

```diff
@@ -70,7 +70,7 @@ std::string ToLower(std::string x) {
 
 
 // Trim whitespace from both ends of string, returning copy of substring
-std::string Trim(const std::string& stringToTrim, const std::string& whitespace)
+std::string Trim(const std::string& stringToTrim, std::string_view whitespace)
```

Comment:
I'm guessing we should start using string_view in a lot of other places and in OP2Utility instead of const std::string& all the time...

Reply:
Probably yes. In some cases it may not matter. There may be a few pitfalls with return values and object lifetime. One interesting benefit is it could increase the possibility of `constexpr` string functions, which could compute stuff at compile time. Certainly some nice to haves.

I'll open an Issue so we can track this and review the code later on for more `string_view` opportunities.